index.js

```diff
@@ -1,4 +1,4 @@
-var spawn = require('child_process').spawn;
+var { spawn, execSync } = require('child_process')
```

Comment:
execSync only works on very new nodes -- does the oldest node for arm support it?